kuma/static/js/analytics.js

```diff
@@ -1,4 +1,15 @@
-(function(win, doc, $) {
+/**
```

Comment:
The changes in this file are to remove jQuery. Note that this file is also used by the wiki site, so if I've screwed anything up, it will also cause problems there.